include/ceedf.h

```diff
@@ -61,25 +73,49 @@ c
 
       integer ceed_eval_weight
       parameter(ceed_eval_weight = 16)
+
 c
 c CeedQualMode
 c
+
       integer ceed_gauss
       parameter(ceed_gauss         = 0)
 
       integer ceed_gauss_lobatto
       parameter(ceed_gauss_lobatto = 1)
 
+c
+c CeedElemTopology
```

Comment:
Fortran 77 does not appear to play nice with unsigned integers, but we only need the unsigned integer on the library side, so I put a tuple with the 'recipe' on the Fortran side that gets converted.

Reply:
Signed and unsigned representations are the same up to `INT_MAX`. I said to use unsigned because right shift has implementation-defined behavior for signed integers. It doesn't really matter which is used in the enum.

Reply:
Note that in Fortran you can write `parameter(CEED_TRIANGLE = X'20001')`.